components/brave_wallet/browser/internal/hd_key_ed25519.cc

```diff
@@ -92,6 +92,11 @@ std::string HDKeyEd25519::GetEncodedPrivateKey() const {
   return GetBase58EncodedKeypair();
```

Comment:
Btw, why GetEncodedPrivateKey() returns a keypair instead of private key?
@darkdh

Reply:
That is how popular Solana wallets like Phantom do, it is always key pair when exporting and extracting private key when importing.

Reply:
Ok, I see.
Can I rename this to something like `GetEncodedPrivateKeyForExport` ?

Reply:
Sure if that makes the function purpose clearer since it is currently the only usage for encoded private key.